jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -406,8 +415,9 @@ void java_bytecode_convert_methodt::convert(
   // (e.g. "my.package.ClassName.myMethodName:(II)I") and query the symbol table
   // to retrieve the symbol (constructed by java_bytecode_convert_method_lazy)
   // associated to the method
-  const irep_idt method_identifier=
-    id2string(class_symbol.name)+"."+id2string(m.name)+":"+m.descriptor;
+  const irep_idt method_identifier =
```

Comment:
I mean not your code, but why do we have a local variable `method_identifier` that then is immediately assigned into `method_id`, perhaps just directly assign?